### ISSUE TITLE
version_table function has hardcoded value for table_name

### ISSUE THREAD

**Issue by PlugaruT** [kvesteri/sqlalchemy-continuum#198]
If you look into the code you can see that the function assumes that the table name will be the default one, `%s_version`. 
If this value if configured when calling function `make_versioned` then there are queries that are failing.
[Here](https://gist.github.com/PlugaruT/28f264ff599b56318c704df6d77ec9c6) is a small script that can reproduce the error.

**Comment by vanife** [comment]
Indeed, the option for `table_name` is not taken into account (https://github.com/kvesteri/sqlalchemy-continuum/blob/master/sqlalchemy_continuum/utils.py#L130), which makes certain use cases fail.

Similarly, default naming convention is hardcoded in : https://github.com/kvesteri/sqlalchemy-continuum/blob/master/sqlalchemy_continuum/dialects/postgresql.py#L482: `table_name[0:-len('_version')],`

**Comment by EricBorczuk** [comment]
@kvesteri Is there any plan to fix this one? We have been using a fork of an old version of SQLAlchemy-Continuum at my company for quite some time now - we want to start using a newer version of it, with the intent of trying native postgres triggers.  Only problem is, in our make_versioned call, we do:

```
make_versioned(
    manager=versioning_manager,
    plugins=[FlaskPlugin()],
    options={'table_name': '%s_versions'}
)
```

And so the above breaks when switching to use Continuum and not our fork.

On our fork, we have a fix for the issue described in `versioned_table` (https://github.com/kvesteri/sqlalchemy-continuum/blob/master/sqlalchemy_continuum/utils.py#L130), which allows make_versioned to work with a custom table_name option.

But we do not currently have a fix for 
sync_triggers (https://github.com/kvesteri/sqlalchemy-continuum/blob/master/sqlalchemy_continuum/dialects/postgresql.py#L482), which would be needed for us to do native versioning, I'd presume.

More than happy to open a PR for the former, or work on the latter and open a PR for both.  Please let me know, and thanks!

**Comment by asandeep** [comment]
+1

I want to override the table name as existing naming convention conflicts with our internal model naming. Any update would be greatly appreciated.
